### ISSUE TITLE
Error when entering world

### ISSUE THREAD

**Issue by eliminer** [FiskFille/TransformersMod#226]
I can't enter world because Minecraft says "Shutting down internal server". Help, please!

**Comment by Gegy** [comment]
Could you provide logs when this error occurs?

**Comment by eliminer** [comment]
Log or crash report?

**Comment by Gegy** [comment]
Log will contain the crash report, if any.

**Comment by eliminer** [comment]
[14:22:32] [main/INFO]: Loading tweak class name cpw.mods.fml.common.launcher.FMLTweaker
[14:22:32] [main/INFO]: Using primary tweak class name cpw.mods.fml.common.launcher.FMLTweaker
[14:22:32] [main/INFO]: Calling tweak class cpw.mods.fml.common.launcher.FMLTweaker
[14:22:32] [main/INFO]: Forge Mod Loader version 7.99.36.1558 for Minecraft 1.7.10 loading
[14:22:32] [main/INFO]: Java is Java HotSpot(TM) 64-Bit Server VM, version 1.8.0_25, running on Windows 8.1:amd64:6.3, installed at C:\Program Files (x86)\Minecraft\runtime\jre-x64\1.8.0_25
[14:22:32] [main/INFO]: Loading tweak class name cpw.mods.fml.common.launcher.FMLInjectionAndSortingTweaker
[14:22:32] [main/INFO]: Loading tweak class name cpw.mods.fml.common.launcher.FMLDeobfTweaker
[14:22:32] [main/INFO]: Calling tweak class cpw.mods.fml.common.launcher.FMLInjectionAndSortingTweaker
[14:22:32] [main/INFO]: Calling tweak class cpw.mods.fml.common.launcher.FMLInjectionAndSortingTweaker
[14:22:32] [main/INFO]: Calling tweak class cpw.mods.fml.relauncher.CoreModManager$FMLPluginWrapper
[14:22:33] [main/INFO]: Found valid fingerprint for Minecraft Forge. Certificate fingerprint e3c3d50c7c986df74c645c0ac54639741c90a557
[14:22:33] [main/INFO]: Found valid fingerprint for Minecraft. Certificate fingerprint cd99959656f753dc28d863b46769f7f8fbaefcfc
[14:22:33] [main/INFO]: Calling tweak class cpw.mods.fml.relauncher.CoreModManager$FMLPluginWrapper
[14:22:33] [main/INFO]: Calling tweak class cpw.mods.fml.relauncher.CoreModManager$FMLPluginWrapper
[14:22:33] [main/INFO]: Calling tweak class cpw.mods.fml.common.launcher.FMLDeobfTweaker
[14:22:33] [main/INFO]: Loading tweak class name cpw.mods.fml.common.launcher.TerminalTweaker
[14:22:33] [main/INFO]: Calling tweak class cpw.mods.fml.common.launcher.TerminalTweaker
[14:22:33] [main/INFO]: Launching wrapped minecraft {net.minecraft.client.main.Main}
[14:22:34] [main/INFO]: Patching Class Entity (sa)
[14:22:34] [main/INFO]: Patching Class Entity done
[14:22:34] [main/INFO]: Patching Class GuiContainer (bex)
[14:22:34] [main/INFO]: Patching Class GuiContainer done
[14:22:34] [main/INFO]: Setting user: eliminer_1
[14:22:35] [Client thread/INFO]: LWJGL Version: 2.9.1
[14:22:36] [Client thread/INFO]: [tv.twitch.StandardCoreAPI:<init>:16]: If on Windows, make sure to provide all of the necessary dll's as specified in the twitchsdk README. Also, make sure to set the PATH environment variable to point to the directory containing the dll's.
[14:22:36] [Client thread/ERROR]: Couldn't initialize twitch stream
[14:22:36] [Client thread/INFO]: Attempting early MinecraftForge initialization
[14:22:36] [Client thread/INFO]: MinecraftForge v10.13.4.1558 Initialized
[14:22:36] [Client thread/INFO]: Replaced 183 ore recipies
[14:22:36] [Client thread/INFO]: Completed early MinecraftForge initialization
[14:22:36] [Client thread/INFO]: Found 0 mods from the command line. Injecting into mod discoverer
[14:22:36] [Client thread/INFO]: Searching C:\Users\Patryk\AppData\Roaming\.minecraft\mods for mods
[14:22:41] [Client thread/INFO]: Forge Mod Loader has identified 5 mods to load
[14:22:42] [Client thread/INFO]: Attempting connection with missing mods [mcp, FML, Forge, transformersg1, transformers] at CLIENT
[14:22:42] [Client thread/INFO]: Attempting connection with missing mods [mcp, FML, Forge, transformersg1, transformers] at SERVER
[14:22:42] [Client thread/INFO]: Reloading ResourceManager: Default, FMLFileResourcePack:Forge Mod Loader, FMLFileResourcePack:Minecraft Forge, FMLFileResourcePack:Transformers Mod G1 Edition, FMLFileResourcePack:Transformers Mod, faithful64pack-v1.4.0.zip
[14:22:42] [Client thread/INFO]: Processing ObjectHolder annotations
[14:22:42] [Client thread/INFO]: Found 341 ObjectHolder annotations
[14:22:42] [Client thread/INFO]: Identifying ItemStackHolder annotations
[14:22:42] [Client thread/INFO]: Found 0 ItemStackHolder annotations
[14:22:42] [Client thread/INFO]: Configured a dormant chunk cache size of 0
[14:22:42] [Client thread/INFO]: Patching Class ModelBiped (bhm)
[14:22:42] [Client thread/INFO]: Patching Class ModelBiped done
[14:22:43] [Client thread/INFO]: Patching Class RenderPlayer (bop)
[14:22:43] [Client thread/INFO]: Patching Class RenderPlayer done
[14:22:43] [Client thread/INFO]: Applying holder lookups
[14:22:43] [Client thread/INFO]: Holder lookups applied
[14:22:43] [Client thread/INFO]: Injecting itemstacks
[14:22:43] [Client thread/INFO]: Itemstack injection complete
[14:22:43] [Sound Library Loader/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: 
[14:22:43] [Sound Library Loader/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: Starting up SoundSystem...
[14:22:43] [Thread-7/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: Initializing LWJGL OpenAL
[14:22:43] [Thread-7/INFO]: [paulscode.sound.SoundSystemLogger:message:69]:     (The LWJGL binding of OpenAL.  For more information, see http://www.lwjgl.org)
[14:22:43] [Thread-7/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: OpenAL initialized.
[14:22:44] [Client thread/INFO]: Created: 16x16 textures/blocks-atlas
[14:22:44] [Client thread/INFO]: Created: 16x16 textures/items-atlas
[14:22:44] [Sound Library Loader/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: 
[14:22:44] [Sound Library Loader/INFO]: Sound engine started
[14:22:44] [Client thread/INFO]: Injecting itemstacks
[14:22:44] [Client thread/INFO]: Itemstack injection complete
[14:22:44] [Client thread/INFO]: Forge Mod Loader has successfully loaded 5 mods
[14:22:44] [Client thread/INFO]: Reloading ResourceManager: Default, FMLFileResourcePack:Forge Mod Loader, FMLFileResourcePack:Minecraft Forge, FMLFileResourcePack:Transformers Mod G1 Edition, FMLFileResourcePack:Transformers Mod, faithful64pack-v1.4.0.zip
[14:22:45] [Client thread/INFO]: Created: 2048x1024 textures/blocks-atlas
[14:22:45] [Client thread/INFO]: Created: 1024x1024 textures/items-atlas
[14:22:46] [Client thread/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: 
[14:22:46] [Client thread/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: SoundSystem shutting down...
[14:22:46] [Client thread/INFO]: [paulscode.sound.SoundSystemLogger:importantMessage:90]:     Author: Paul Lamb, www.paulscode.com
[14:22:46] [Client thread/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: 
[14:22:46] [Sound Library Loader/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: 
[14:22:46] [Sound Library Loader/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: Starting up SoundSystem...
[14:22:46] [Thread-9/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: Initializing LWJGL OpenAL
[14:22:46] [Thread-9/INFO]: [paulscode.sound.SoundSystemLogger:message:69]:     (The LWJGL binding of OpenAL.  For more information, see http://www.lwjgl.org)
[14:22:46] [Thread-9/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: OpenAL initialized.
[14:22:46] [Sound Library Loader/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: 
[14:22:46] [Sound Library Loader/INFO]: Sound engine started
[14:22:53] [Server thread/INFO]: Starting integrated minecraft server version 1.7.10
[14:22:53] [Server thread/INFO]: Generating keypair
[14:22:53] [Server thread/INFO]: Injecting existing block and item data into this server instance
[14:22:53] [Server thread/INFO]: Applying holder lookups
[14:22:53] [Server thread/INFO]: Holder lookups applied
[14:22:53] [Server thread/INFO]: Loading dimension 0 (New p) (net.minecraft.server.integrated.IntegratedServer@745aebb3)
[14:22:53] [Server thread/INFO]: Loading dimension 1 (New p) (net.minecraft.server.integrated.IntegratedServer@745aebb3)
[14:22:54] [Server thread/INFO]: Loading dimension -1 (New p) (net.minecraft.server.integrated.IntegratedServer@745aebb3)
[14:22:54] [Server thread/INFO]: Preparing start region for level 0
[14:22:55] [Server thread/INFO]: Preparing spawn area: 72%
[14:22:55] [Server thread/INFO]: Changing view distance to 32, from 10
[14:22:56] [Netty Client IO #0/INFO]: Server protocol version 2
[14:22:56] [Netty IO #1/INFO]: Client protocol version 2
[14:22:56] [Netty IO #1/INFO]: Client attempting to join with 5 mods : transformers@0.6.3,FML@7.10.99.99,transformersg1@1.1.1,Forge@10.13.4.1558,mcp@9.05
[14:22:56] [Netty IO #1/INFO]: Attempting connection with missing mods [] at CLIENT
[14:22:56] [Netty Client IO #0/INFO]: Attempting connection with missing mods [] at SERVER
[14:22:56] [Server thread/INFO]: [Server thread] Server side modded connection established
[14:22:56] [Client thread/INFO]: [Client thread] Client side modded connection established
[14:22:56] [Server thread/INFO]: eliminer_1[local:E:11c405f6] logged in with entity id 485 at (-1.5, 72.0, 260.5)
[14:22:56] [Server thread/INFO]: eliminer_1 joined the game
[14:23:04] [Server thread/INFO]: Stopping server

**Comment by eliminer** [comment]
Correct?

**Comment by eliminer** [comment]
Sorry. Bad one.

**Comment by eliminer** [comment]
[10:28:46] [main/INFO]: Loading tweak class name cpw.mods.fml.common.launcher.FMLTweaker
[10:28:46] [main/INFO]: Using primary tweak class name cpw.mods.fml.common.launcher.FMLTweaker
[10:28:46] [main/INFO]: Calling tweak class cpw.mods.fml.common.launcher.FMLTweaker
[10:28:46] [main/INFO]: Forge Mod Loader version 7.99.36.1558 for Minecraft 1.7.10 loading
[10:28:46] [main/INFO]: Java is Java HotSpot(TM) 64-Bit Server VM, version 1.8.0_25, running on Windows 8.1:amd64:6.3, installed at C:\Program Files (x86)\Minecraft\runtime\jre-x64\1.8.0_25
[10:28:46] [main/INFO]: Loading tweak class name cpw.mods.fml.common.launcher.FMLInjectionAndSortingTweaker
[10:28:46] [main/INFO]: Loading tweak class name cpw.mods.fml.common.launcher.FMLDeobfTweaker
[10:28:46] [main/INFO]: Calling tweak class cpw.mods.fml.common.launcher.FMLInjectionAndSortingTweaker
[10:28:46] [main/INFO]: Calling tweak class cpw.mods.fml.common.launcher.FMLInjectionAndSortingTweaker
[10:28:46] [main/INFO]: Calling tweak class cpw.mods.fml.relauncher.CoreModManager$FMLPluginWrapper
[10:28:47] [main/INFO]: Found valid fingerprint for Minecraft Forge. Certificate fingerprint e3c3d50c7c986df74c645c0ac54639741c90a557
[10:28:47] [main/INFO]: Found valid fingerprint for Minecraft. Certificate fingerprint cd99959656f753dc28d863b46769f7f8fbaefcfc
[10:28:47] [main/INFO]: Calling tweak class cpw.mods.fml.relauncher.CoreModManager$FMLPluginWrapper
[10:28:47] [main/INFO]: Calling tweak class cpw.mods.fml.relauncher.CoreModManager$FMLPluginWrapper
[10:28:47] [main/INFO]: Calling tweak class cpw.mods.fml.common.launcher.FMLDeobfTweaker
[10:28:47] [main/INFO]: Loading tweak class name cpw.mods.fml.common.launcher.TerminalTweaker
[10:28:47] [main/INFO]: Calling tweak class cpw.mods.fml.common.launcher.TerminalTweaker
[10:28:47] [main/INFO]: Launching wrapped minecraft {net.minecraft.client.main.Main}
[10:28:48] [main/INFO]: Patching Class Entity (sa)
[10:28:48] [main/INFO]: Patching Class Entity done
[10:28:48] [main/INFO]: Patching Class GuiContainer (bex)
[10:28:48] [main/INFO]: Patching Class GuiContainer done
[10:28:48] [main/INFO]: Setting user: eliminer_1
[10:28:49] [Client thread/INFO]: LWJGL Version: 2.9.1
[10:28:50] [Client thread/INFO]: [tv.twitch.StandardCoreAPI:<init>:16]: If on Windows, make sure to provide all of the necessary dll's as specified in the twitchsdk README. Also, make sure to set the PATH environment variable to point to the directory containing the dll's.
[10:28:50] [Client thread/ERROR]: Couldn't initialize twitch stream
[10:28:50] [Client thread/INFO]: Attempting early MinecraftForge initialization
[10:28:50] [Client thread/INFO]: MinecraftForge v10.13.4.1558 Initialized
[10:28:50] [Client thread/INFO]: Replaced 183 ore recipies
[10:28:50] [Client thread/INFO]: Completed early MinecraftForge initialization
[10:28:50] [Client thread/INFO]: Found 0 mods from the command line. Injecting into mod discoverer
[10:28:50] [Client thread/INFO]: Searching C:\Users\Patryk\AppData\Roaming\.minecraft\mods for mods
[10:28:55] [Client thread/INFO]: Forge Mod Loader has identified 5 mods to load
[10:28:55] [Client thread/INFO]: Attempting connection with missing mods [mcp, FML, Forge, transformersg1, transformers] at CLIENT
[10:28:55] [Client thread/INFO]: Attempting connection with missing mods [mcp, FML, Forge, transformersg1, transformers] at SERVER
[10:28:56] [Client thread/INFO]: Reloading ResourceManager: Default, FMLFileResourcePack:Forge Mod Loader, FMLFileResourcePack:Minecraft Forge, FMLFileResourcePack:Transformers Mod G1 Edition, FMLFileResourcePack:Transformers Mod, faithful64pack-v1.4.0.zip
[10:28:56] [Client thread/INFO]: Processing ObjectHolder annotations
[10:28:56] [Client thread/INFO]: Found 341 ObjectHolder annotations
[10:28:56] [Client thread/INFO]: Identifying ItemStackHolder annotations
[10:28:56] [Client thread/INFO]: Found 0 ItemStackHolder annotations
[10:28:56] [Client thread/INFO]: Configured a dormant chunk cache size of 0
[10:28:57] [Client thread/INFO]: Patching Class ModelBiped (bhm)
[10:28:57] [Client thread/INFO]: Patching Class ModelBiped done
[10:28:57] [Client thread/INFO]: Patching Class RenderPlayer (bop)
[10:28:57] [Client thread/INFO]: Patching Class RenderPlayer done
[10:28:57] [Client thread/INFO]: Applying holder lookups
[10:28:57] [Client thread/INFO]: Holder lookups applied
[10:28:57] [Client thread/INFO]: Injecting itemstacks
[10:28:57] [Client thread/INFO]: Itemstack injection complete
[10:28:57] [Sound Library Loader/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: 
[10:28:57] [Sound Library Loader/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: Starting up SoundSystem...
[10:28:57] [Thread-7/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: Initializing LWJGL OpenAL
[10:28:57] [Thread-7/INFO]: [paulscode.sound.SoundSystemLogger:message:69]:     (The LWJGL binding of OpenAL.  For more information, see http://www.lwjgl.org)
[10:28:57] [Thread-7/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: OpenAL initialized.
[10:28:58] [Sound Library Loader/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: 
[10:28:58] [Sound Library Loader/INFO]: Sound engine started
[10:28:58] [Client thread/INFO]: Created: 16x16 textures/blocks-atlas
[10:28:58] [Client thread/INFO]: Created: 16x16 textures/items-atlas
[10:28:58] [Client thread/INFO]: Injecting itemstacks
[10:28:58] [Client thread/INFO]: Itemstack injection complete
[10:28:58] [Client thread/INFO]: Forge Mod Loader has successfully loaded 5 mods
[10:28:58] [Client thread/INFO]: Reloading ResourceManager: Default, FMLFileResourcePack:Forge Mod Loader, FMLFileResourcePack:Minecraft Forge, FMLFileResourcePack:Transformers Mod G1 Edition, FMLFileResourcePack:Transformers Mod, faithful64pack-v1.4.0.zip
[10:29:00] [Client thread/INFO]: Created: 2048x1024 textures/blocks-atlas
[10:29:00] [Client thread/INFO]: Created: 1024x1024 textures/items-atlas
[10:29:00] [Client thread/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: 
[10:29:00] [Client thread/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: SoundSystem shutting down...
[10:29:00] [Client thread/INFO]: [paulscode.sound.SoundSystemLogger:importantMessage:90]:     Author: Paul Lamb, www.paulscode.com
[10:29:00] [Client thread/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: 
[10:29:00] [Sound Library Loader/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: 
[10:29:00] [Sound Library Loader/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: Starting up SoundSystem...
[10:29:00] [Thread-9/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: Initializing LWJGL OpenAL
[10:29:00] [Thread-9/INFO]: [paulscode.sound.SoundSystemLogger:message:69]:     (The LWJGL binding of OpenAL.  For more information, see http://www.lwjgl.org)
[10:29:00] [Thread-9/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: OpenAL initialized.
[10:29:01] [Sound Library Loader/INFO]: [paulscode.sound.SoundSystemLogger:message:69]: 
[10:29:01] [Sound Library Loader/INFO]: Sound engine started
[10:29:16] [Client thread/INFO]: Deleting level New p
[10:29:16] [Client thread/INFO]: Attempt 1...
[10:29:25] [Server thread/INFO]: Starting integrated minecraft server version 1.7.10
[10:29:25] [Server thread/INFO]: Generating keypair
[10:29:25] [Server thread/INFO]: Converting map!
[10:29:25] [Server thread/INFO]: Scanning folders...
[10:29:25] [Server thread/INFO]: Total conversion count is 0
[10:29:25] [Server thread/INFO]: Injecting existing block and item data into this server instance
[10:29:25] [Server thread/INFO]: Applying holder lookups
[10:29:25] [Server thread/INFO]: Holder lookups applied
[10:29:25] [Server thread/WARN]: Unable to find spawn biome
[10:29:26] [Server thread/INFO]: Loading dimension 0 ([ovhfg) (net.minecraft.server.integrated.IntegratedServer@5c393e13)
[10:29:26] [Server thread/INFO]: Loading dimension 1 ([ovhfg) (net.minecraft.server.integrated.IntegratedServer@5c393e13)
[10:29:26] [Server thread/INFO]: Loading dimension -1 ([ovhfg) (net.minecraft.server.integrated.IntegratedServer@5c393e13)
[10:29:26] [Server thread/INFO]: Preparing start region for level 0
[10:29:27] [Server thread/INFO]: Preparing spawn area: 20%
[10:29:28] [Server thread/INFO]: Preparing spawn area: 45%
[10:29:29] [Server thread/INFO]: Preparing spawn area: 69%
[10:29:30] [Server thread/INFO]: Preparing spawn area: 95%
[10:29:30] [Server thread/INFO]: Changing view distance to 32, from 10
[10:29:31] [Netty Client IO #0/INFO]: Server protocol version 2
[10:29:31] [Netty IO #1/INFO]: Client protocol version 2
[10:29:31] [Netty IO #1/INFO]: Client attempting to join with 5 mods : transformers@0.6.3,FML@7.10.99.99,transformersg1@1.1.1,Forge@10.13.4.1558,mcp@9.05
[10:29:31] [Netty IO #1/INFO]: Attempting connection with missing mods [] at CLIENT
[10:29:31] [Netty Client IO #0/INFO]: Attempting connection with missing mods [] at SERVER
[10:29:31] [Server thread/INFO]: [Server thread] Server side modded connection established
[10:29:31] [Server thread/INFO]: eliminer_1[local:E:b43134ee] logged in with entity id 240 at (311.5, 70.0, -552.5)
[10:29:31] [Client thread/INFO]: [Client thread] Client side modded connection established
[10:29:31] [Server thread/INFO]: eliminer_1 joined the game
[10:29:31] [Server thread/INFO]: eliminer_1 has just earned the achievement [Transformers Mod]
[10:29:39] [Server thread/INFO]: Stopping server
[10:29:39] [Server thread/INFO]: Saving players
[10:29:39] [Server thread/INFO]: Saving worlds
[10:29:39] [Server thread/INFO]: Saving chunks for level '[ovhfg'/Overworld
[10:30:03] [Server thread/INFO]: Saving chunks for level '[ovhfg'/Nether
[10:30:03] [Server thread/INFO]: Saving chunks for level '[ovhfg'/The End
[10:30:04] [Server thread/INFO]: Unloading dimension 0
[10:30:04] [Server thread/INFO]: Unloading dimension -1
[10:30:04] [Server thread/INFO]: Unloading dimension 1
[10:30:05] [Server thread/INFO]: Applying holder lookups
[10:30:05] [Server thread/INFO]: Holder lookups applied
[10:30:05] [Client thread/FATAL]: Unreported exception thrown!
java.lang.IndexOutOfBoundsException
	at java.nio.Buffer.checkIndex(Buffer.java:540) ~[?:1.8.0_25]
	at java.nio.DirectIntBufferU.get(DirectIntBufferU.java:253) ~[?:1.8.0_25]
	at net.minecraft.client.renderer.RenderGlobal.func_72712_a(RenderGlobal.java:350) ~[bma.class:?]
	at net.minecraft.client.renderer.RenderGlobal.func_72732_a(RenderGlobal.java:294) ~[bma.class:?]
	at net.minecraft.client.Minecraft.func_71353_a(Minecraft.java:2216) ~[bao.class:?]
	at net.minecraft.client.Minecraft.func_71403_a(Minecraft.java:2146) ~[bao.class:?]
	at net.minecraft.client.network.NetHandlerPlayClient.func_147282_a(NetHandlerPlayClient.java:240) ~[bjb.class:?]
	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:70) ~[hd.class:?]
	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:13) ~[hd.class:?]
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212) ~[ej.class:?]
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:2061) ~[bao.class:?]
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973) ~[bao.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898) [bao.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:148) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_25]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_25]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_25]
	at java.lang.reflect.Method.invoke(Method.java:483) ~[?:1.8.0_25]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
[10:30:05] [Client thread/INFO]: [net.minecraft.client.Minecraft:func_71377_b:349]: ---- Minecraft Crash Report ----
// My bad.

Time: 18.08.17 10:30
Description: Unexpected error

java.lang.IndexOutOfBoundsException
	at java.nio.Buffer.checkIndex(Buffer.java:540)
	at java.nio.DirectIntBufferU.get(DirectIntBufferU.java:253)
	at net.minecraft.client.renderer.RenderGlobal.func_72712_a(RenderGlobal.java:350)
	at net.minecraft.client.renderer.RenderGlobal.func_72732_a(RenderGlobal.java:294)
	at net.minecraft.client.Minecraft.func_71353_a(Minecraft.java:2216)
	at net.minecraft.client.Minecraft.func_71403_a(Minecraft.java:2146)
	at net.minecraft.client.network.NetHandlerPlayClient.func_147282_a(NetHandlerPlayClient.java:240)
	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:70)
	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:13)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:2061)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at java.nio.Buffer.checkIndex(Buffer.java:540)
	at java.nio.DirectIntBufferU.get(DirectIntBufferU.java:253)
	at net.minecraft.client.renderer.RenderGlobal.func_72712_a(RenderGlobal.java:350)
	at net.minecraft.client.renderer.RenderGlobal.func_72732_a(RenderGlobal.java:294)
	at net.minecraft.client.Minecraft.func_71353_a(Minecraft.java:2216)
	at net.minecraft.client.Minecraft.func_71403_a(Minecraft.java:2146)
	at net.minecraft.client.network.NetHandlerPlayClient.func_147282_a(NetHandlerPlayClient.java:240)
	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:70)
	at net.minecraft.network.play.server.S01PacketJoinGame.func_148833_a(SourceFile:13)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)

-- Affected level --
Details:
	Level name: MpServer
	All players: 0 total; []
	Chunk stats: MultiplayerChunkCache: 0, 0
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: 
	Level spawn location: World: (8,64,8), Chunk: (at 8,4,8 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 0 game time, 0 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
	Forced entities: 0 total; []
	Retry entities: 0 total; []
	Server brand: ~~ERROR~~ NullPointerException: null
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:373)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2444)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:927)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 8.1 (amd64) version 6.3
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 456571816 bytes (435 MB) / 1950248960 bytes (1859 MB) up to 8375238656 bytes (7987 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx8G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn2048M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1558 5 mods loaded, 5 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1558-1.7.10.jar) 
	UCHIJAAAA	Forge{10.13.4.1558} [Minecraft Forge] (forge-1.7.10-10.13.4.1558-1.7.10.jar) 
	UCHIJAAAA	transformers{0.6.3} [Transformers Mod] (TransformersMod-1.7.10-0.6.3.jar) 
	UCHIJAAAA	transformersg1{1.1.1} [Transformers Mod G1 Edition] (TransformersG1-1.7.10-1.1.1.jar) 
	Launched Version: 1.7.10-Forge10.13.4.1558-1.7.10
	LWJGL: 2.9.1
	OpenGL: Intel(R) HD Graphics 4600 GL version 4.3.0 - Build 10.18.14.4062, Intel
	GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: [faithful64pack-v1.4.0.zip]
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Anisotropic Filtering: Off (1)
[10:30:05] [Client thread/INFO]: [net.minecraft.client.Minecraft:func_71377_b:359]: #@!@# Game crashed! Crash report saved to: #@!@# C:\Users\Patryk\AppData\Roaming\.minecraft\crash-reports\crash-2017-08-18_10.30.05-client.txt

**Comment by Gegy** [comment]
Unrelated, you have your render distance set to something > 16, and 1.7.10 does not support that.

**Comment by FiskFille** [comment]
In layman's terms, just turn down your render distance and you'll be fine.
